frontend-project/.prettierrc.js

```diff
@@ -2,4 +2,5 @@ const fabric = require('@umijs/fabric');
 
 module.exports = {
   ...fabric.prettier,
+  arrowParens: 'avoid',
```

Comment:
See https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev .

In following PR we can drop that line.